src/term_rst_post/ansimotd.py

```diff
@@ -65,7 +65,7 @@ def accomodate_motd(body_file, head_file=None, foot_file=None, foot_link=None, w
 
     # Inject extra link between body and footer
     if foot_link:
-        part_link = f"\nMore information in {foot_link}\n"
+        part_link = f"\nMore information in\n{foot_link}\n"
```

Comment:
can we make it give an error if the link is longer then the wrapping size?

Reply:
As long as the link can be copy pasted into the browser and it works, the behaviour is correct. I would not try to support clickable links in terminals beyond minimizing the occurrences of wrapped links, that is not standard a standard feature.

Reply:
It must be noted that the same issue can appear in the body of the MOTD. If there is long URL in the text, it will be wrapped. And in that case it is much more difficult to detect those links.

Reply:
why not increase the line width to say 100? 70 characters is really small..